src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const listProperties = document.querySelectorAll('li');
+const nameList = [...listProperties].map(
+  item => item.innerText);
+const salaryList = [...listProperties].map(
+  item => item.getAttribute('data-salary'));
+const ageList = [...listProperties].map(
+  item => item.getAttribute('data-age'));
+const positionList = [...listProperties].map(
+  item => item.getAttribute('data-position'));
+const list = [];
+
+for (let i = 0; i < nameList.length; i++) {
+  const obj = {};
+
+  obj.name = nameList[i];
+  obj.position = positionList[i];
```

Comment:
```suggestion
  const obj = {
     position: listProperties[i].dataset.position,
     salary: listProperties[i].dataset.salary,
     ... і т.д.
  }
```
Ось так набагато лаконічніше, і не потрібно створювати купу лишніх лістів)